### ISSUE TITLE
Optimizing startup time

### ISSUE THREAD

**Issue by me21** [zauberzeug/nicegui#37]
It would be nice to conditionally omit imports of LinePlot and Plot classes. Doing so brought the startup time on Beaglebone Black single board computer from 20+ to 8 seconds.

Another similar thing is justpy importing its chartcomponents.py inside, but perhaps this is an issue for their own repository.

**Comment by falkoschindler** [comment]
I'm having trouble reproducing your result. The following code measures the time from starting the script until the UI is running. Independent of whether LinePlot and Plot are imported in `ui.py`, the time is always around 1.4s on my MacBook. Maybe it's an issue with Matplotlib on different processor architectures. Or I misunderstood something.

```python
import time
t = time.time()

from nicegui import ui

ui.label('Hello world!')

ui.on_startup(lambda: print(f'{time.time() - t:.3f}'))

ui.run(port=1234)
```

**Comment by me21** [comment]
I'll try to run your code, but I think MacBook is much more powerful than Beaglebone Black, so maybe it's not noticeable there.
I'll update this issue later.

**Comment by me21** [comment]
With commented imports:

```
debian@beaglebone:~$ web_env/bin/python nicegui_test.py
/home/debian/web_env/lib/python3.7/site-packages/justpy
Module directory: /home/debian/web_env/lib/python3.7/site-packages/justpy, Application directory: /home/debian
JustPy ready to go on http://0.0.0.0:1234
14.747
```

With uncommented imports:

```
debian@beaglebone:~$ web_env/bin/python nicegui_test.py
/home/debian/web_env/lib/python3.7/site-packages/justpy
Module directory: /home/debian/web_env/lib/python3.7/site-packages/justpy, Application directory: /home/debian
JustPy ready to go on http://0.0.0.0:1234
37.066
```

**Comment by falkoschindler** [comment]
Ok, it is finally released as [v0.7.29](https://github.com/zauberzeug/nicegui/releases/tag/v0.7.29)
